trlx/models/modeling_ilql.py

```diff
@@ -339,3 +353,147 @@ def post_init(self, state_dict):
         self.ilql_heads.load_state_dict(state_dict, strict=False)
         del state_dict
         gc.collect()
+
+
+class AutoModelForSeq2SeqLMWithILQLHeads(PreTrainedModelWrapper):
```

Comment:
Let's add a test for this similar to https://github.com/CarperAI/trlx/blob/e72f7d1a8008c9a994e9fe465aa4a8a7a1fb3232/tests/test_models.py#L138